bookShop/src/app/app.component.html

```diff
@@ -1,6 +1,9 @@
 <h1 #appTitle class="title"></h1>
 
-<app-book-component (buyEvent)="onBuy($event)"></app-book-component>
-<app-cart-component></app-cart-component>
+<nav>
+  <a routerLink="">Main page</a>
```

Comment:
You can add `routerLinkActive` to append some styles for active link

Reply:
Fix